gammapy/scripts/tests/test_sherpa_like.py

```diff
@@ -0,0 +1,51 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import json
+import os
+
+from astropy.io import fits
+
+from ...datasets import load_poisson_stats_image
+from ..sherpa_like import sherpa_image_like
+from numpy.testing.utils import assert_allclose
+
+
```

Comment:
You have to mark the test as using the optional Sherpa dependency:
https://travis-ci.org/gammapy/gammapy/jobs/77172790#L1154
http://astropy.readthedocs.org/en/latest/development/testguide.html#tests-requiring-optional-dependencies

And please add a changelog entry for this.